src/Makefile.nds.arm9

```diff
@@ -102,10 +105,11 @@ export INCLUDE	:=	$(foreach dir,$(INCLUDES),-I$(CURDIR)/$(dir)) \
 export LIBPATHS	:=	$(foreach dir,$(LIBDIRS),-L$(dir)/lib)
  
 .PHONY: $(BUILD) clean
- 
+
 #---------------------------------------------------------------------------------
 $(BUILD):
 	@[ -d $@ ] || mkdir -p $@
+	@[ -d $@/borg] || mkdir -p $@/borg
```

Comment:
That's right, but what I gave you was wrong:  there needs to be a space between "borg" and "]".  That'll get rid of the "bash: line 0: [: missing `]'" message in the build results.  A similar change is needed for src/Makefile.nds.arm9.